libsrc/apple2/get_tv.s

```diff
@@ -0,0 +1,151 @@
+        .export     _get_tv
+
+        .import     _set_iigs_speed, _get_iigs_speed
+        .import     _bzero, _memset, pushax
+        .import     ostype
+
+        .importzp   tmp1, ptr1
+
+        .constructor calibrate_tv, 2
+
+        .include    "apple2.inc"
+        .include    "accelerator.inc"
+
+.segment "ONCE"
+
+; Cycle wasters
+waste_72:
+        jsr     waste_36
+waste_36:
+        jsr     waste_12
+waste_24:
+        jsr     waste_12
+waste_12:
+        rts
+
+.proc calibrate_tv
```

Comment:
A general remark: You may have noticed that there was (nearly) no .proc inside the apple2 directory before you started to contribute to it. That was because you have not too seldomly two functions share code by one falling through into the other. So for consistency reasons I personally stayed clear from .proc altogether.

Reply:
Ah, that's why. I like the .procs because it gives my debugger the size of the function. In my own code I have some fallthroughs that I comment this way:

```
.proc extra_things
...
        ; Fallthrough to things
.endproc

.proc things
...
```

I can remove them, though.